capi/src/layout_editor.rs

```diff
@@ -4,6 +4,7 @@
 //! state objects that can be visualized by any kind of User Interface.
 
 use super::{output_vec, Json};
+use crate::layout_editor_state::OwnedLayoutEditorState;
```

Comment:
Seems like cargo fmt wants this reordered, doesn't matter too much though.

Reply:
I meant to run `cargo fmt` before opening the PR. Whoops.